homeassistant/components/cover/__init__.py

```diff
@@ -187,6 +187,8 @@ class CoverEntity(Entity):
     _attr_is_opening: bool | None = None
     _attr_state: None = None
 
+    _attr_is_last_toggle_direction_open = True
```

Comment:
Let's not call this `_attr_-something` to not confuse it with an entity attribute.
Maybe name it `_cover_is_last_toggle_direction_open` instead.

Reply:
Done!